helper.php

```diff
@@ -412,6 +413,7 @@ public function printActionTranslatePage() {
                                              '','',array('readonly'=>'readonly')));
                                              
         $form->addElement(form_makeTextField('title',$target_title,$this->getLang('translated_title'),'',$class));
+        $form->addElement(form_makeTextField('target_id',$target_id,$this->getLang('translated_id'),'',$class));
         $form->addElement(form_makeButton('submit','', $this->getLang('create_translation')));
```

Comment:
I would like some text that says that the page id is optional and an automatic id will be generated if it is ommitted. Personally I think it's a good rule that the page id always matches the first heading on the page, like in mediawiki (e.g. wikipedia) and that the form can have as few fields as possible.

What would you think about a setting checkbox "let translators choose page id" or something like that?